src/main/java/dev/openfeature/sdk/ClientMetadata.java

```diff
@@ -0,0 +1,10 @@
+package dev.openfeature.sdk;
+
+/**
+ * Holds identifying information about a Client.
+ */
+public interface ClientMetadata {
```

Comment:
We should avoid this and use `Metadata` interface with new non-breaking default method :)